modules/calcite/src/test/java/org/apache/ignite/internal/processors/query/calcite/integration/StdSqlOperatorsTest.java

```diff
@@ -285,6 +285,8 @@ public void testCollections() {
         assertExpression("CARDINALITY(ARRAY[1, 2, 3])").returns(3).check();
         assertExpression("ARRAY[1, 2, 3] IS EMPTY").returns(false).check();
         assertExpression("ARRAY[1, 2, 3] IS NOT EMPTY").returns(true).check();
+        assertQuery("SELECT * FROM UNNEST(ARRAY[1, 2]) WITH ORDINALITY").returns(1, 1).returns(2, 2)
```

Comment:
and without the ordinality?

Reply:
There are a lot of tests for unnest with/without ordinality in UnnestIntegrationTest. StdSqlOperatorsTest is a smoke test for operators. It contains one test per each operator to not overload class.